### ISSUE TITLE
PayPal Standard not an available payment option when Reference Transactions are enabled for the PayPal account

### ISSUE THREAD

**Issue by jrick1229** [woocommerce/woocommerce-gutenberg-products-block#4242]
## Describe the bug

When _Reference Transactions_ are enabled on a merchant's PayPal account the payment method does not show as an option in checkout on a checkout block.

## To reproduce

1. Enable PayPal Standard **without** Reference Transactions
2. Add a product to the cart
3. Navigate to the checkout block
4. You'll see PayPal Standard as a payment option: 
5. Switch to a PayPal merchant account **with** Reference Transactions enabled
6. Navigate back to the checkout block
7. You'll see that PayPal Standard **is not** a possible payment option

If you already have an account with Reference Transactions enabled, you can use this plugin to quickly disable Reference Transactions for testing on your account: https://github.com/woocommerce/woocommerce-subscriptions-disable-paypal-reference-transactions

## Expected behavior

While Reference Transactions does use a slightly different way of accepting payments (don't believe it uses IPN), I would still expect it to show as a possible payment method.

## Screenshots

Without Reference Transactions enabled: https://d.pr/i/DhefyN
With Reference Transactions enabled: https://d.pr/i/VUDUrB

### WordPress (please complete the following information):

<details>

```
### WordPress Environment ###

WordPress address (URL): http://testsite10.local
Site address (URL): http://testsite10.local
WC Version: 5.3.0
REST API Version: ✔ 5.3.0
WC Blocks Version: ✔ 5.1.0
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 2.3.0
Log Directory Writable: ✔
WP Version: 5.7.2

### Database ###

WC Database Version: 5.3.0

### Active Plugins (2) ###

WooCommerce Blocks: by Automattic – 5.1.0
WooCommerce: by Automattic – 5.3.0

### Theme ###

Name: Storefront
Version: 3.5.1 (update to version 3.6.0 is available)
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Payment Gateway Support ###

Stripe SEPA Direct Debit: products
refunds
tokenization
add_payment_method
subscriptions
subscription_cancellation
subscription_suspension
subscription_reactivation
subscription_amount_changes
subscription_date_changes
subscription_payment_method_change
subscription_payment_method_change_customer
subscription_payment_method_change_admin
multiple_subscriptions
pre-orders

PayPal Standard: subscription_payment_method_change_customer
subscription_payment_method_change_admin
subscription_amount_changes
subscription_date_changes
multiple_subscriptions
subscription_payment_method_delayed_change
subscriptions
subscription_cancellation
subscription_suspension
subscription_reactivation
products
refunds

PayPal Reference Transactions Enabled: ✔ Yes
```
</details>

## Additional context

4002619-zen

**Comment by Levi-70** [comment]
@nerrad, @jrick1229 @james-allan  just curious was this issue fixed because we are having the exact problem

**Comment by opr** [comment]
Howdy, @derher and @jrick1229 - a new version of [WooCommerce Subscriptions](https://woocommerce.com/products/woocommerce-subscriptions/) was released earlier today, this release contains the fix for this issue.

**Comment by Levi-70** [comment]
Thanks a lot @opr. is there any time able on when it will be released (for me to be able to update)

**Comment by opr** [comment]
Hey @derher - just to clarify - the cause of this issue was the WooCommerce Blocks integration in WooCommerce Subscriptions, so if you've got a subscription to WooCommerce Subscriptions, you should be able to download the latest version from your WooCommerce account. https://woocommerce.com/my-account/downloads/ the version containing the fix is version 3.1.3 

If you don't have a current active subscription to this product, you'll need to buy one from this page: https://woocommerce.com/products/woocommerce-subscriptions/

If you need any further help with this please get in touch with our fantastic support team here and they'll be able to assist you further: https://woocommerce.com/contact-us/

**Comment by Levi-70** [comment]
ok i see it shows up now in the plugins page
thanks a lot
